install_sct

```diff
@@ -335,7 +345,8 @@ for arg in "$@"; do
   esac
 done
 
-while getopts ":dhbpv" opt; do
+NONINTERACTIVE=""
+while getopts ":dhbpvy" opt; do
```

Comment:
`-p` isn't used in the case statement, is it?

```suggestion
while getopts ":dhbvy" opt; do
```

Reply:
oh good catch!